scripts/filestash/create_config.py

```diff
@@ -19,6 +17,23 @@
 CONFIG_JSON = Path(tempfile.mkdtemp()) / "filestash_config.json"
 
 
+# The distutils package has been deprecated in Python 3.10, so I copied the
+# strtobool function from the distutils.util module.
+def strtobool(val):
+    """Convert a string representation of truth to true (1) or false (0).
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+    """
+    val = val.lower()
+    if val in ("y", "yes", "t", "true", "on", "1"):
```

Comment:
For this we normally use pydantic parser since it already implements all these variations 

```python
from pydantic import parse_obj_as

v = parse_obj_as(bool, value)
```

```cmd
>>> from pydantic import parse_obj_as
>>> parse_obj_as(bool, "yes")
True
>>> parse_obj_as(bool, "no")
False
>>> parse_obj_as(bool, "n")
False
>>> parse_obj_as(bool, "1")
True
>>> parse_obj_as(bool, "0")
False
```

Reply:
Generally, I agree, but in this special case, I would leave it as is so we do not need to introduce pydantic dependency in the basic devenv.